audiomentations/augmentations/bit_crush.py

```diff
@@ -28,8 +28,10 @@ def __init__(self, min_bit_depth: int = 5, max_bit_depth: int = 10, p: float = 0
     def randomize_parameters(self, samples: NDArray[np.float32], sample_rate: int):
         super().randomize_parameters(samples, sample_rate)
         if self.parameters["should_apply"]:
-            self.parameters["bit_depth"] = random.uniform(
-                self.min_bit_depth, self.max_bit_depth
+            self.parameters["bit_depth"] = int(
+                random.uniform(
+                    self.min_bit_depth, self.max_bit_depth
+                )
             )
```

Comment:
using random.randint() without an int() around it would be even better, because that would include the max_bit_depth

Reply:
Nice suggestion. Done!